benchmark/benchmarks.jl

```diff
@@ -1,10 +1,20 @@
 using BenchmarkTools
 using ParticleDA
+using MPI
 
 include(joinpath(joinpath(@__DIR__, "..", "test"), "model", "model.jl"))
 using .Model
 
+if !MPI.Initialized()
+    MPI.Init()
+end
+
 const SUITE = BenchmarkGroup()
+const my_rank = MPI.Comm_rank(MPI.COMM_WORLD)
+const my_size = MPI.Comm_size(MPI.COMM_WORLD)
+
+SUITE["base"] = BenchmarkGroup()
+SUITE["BootstrapFilter"] = BenchmarkGroup()
```

Comment:
Now we have these two benchmark groups: `"base"` is common to all filters (I think, let me know if I missed something!), `"BootstrapFilter"` runs benchmarks for `init_filter` and `run_particle_filter` which in #133 will have different implementations for the optimal filters

Reply:
What about `update_particle_dynamics!`?

Reply:
Good catch!